doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1714,7 +1648,7 @@ variety of commands:
    This warning is an error by default. It is caused by using a
    command with some attribute it does not understand.
 
-.. [1]
+.. comment [1]
```

Comment:
The footnote [2] remains numbered 2, which is a bit weird. Can we fix this? I.e. make the only remaining footnote be numbered 1?

Reply:
Got it.

Seems a bit primitive that the author has to manage numbering the footnotes.

Reply:
I don't know if there are better ways to do this.

Reply:
See https://docutils.sourceforge.io/docs/user/rst/quickref.html#footnotes ; You can use [#] as the number for autonumbering